### PR TITLE
Review

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
### Project Requirements

Our task is to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.

The Space Travelers' Hub consists of Rockets, Missions, and the My Profile section.

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
All linter checks have passed.
No merge conflicts.

File: src/components/Profile.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  let count = 0;
+  return (
+    <div className="row">
+      <div className="col-5 offset-1">
+        <MyProfile />
+      </div>
+      <div className="col-5">
+        <h1>My Rockets</h1>
+        <div>
+          {/* eslint-disable-next-line */}
```
Comment (bimbolabuari) [inline src/components/Profile.js]: Kindly remove this disabled linter, linting improves readability, removes errors. 👍

Please check for other occurrences 🙂

File: README.md [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A website in which customers can reserve rockets and mission of space-X
+
+### **Student A**
+
+Chuanqi Zhu
+
+### **Student B**
+
+Stephenia Ugboaja Uchechi
```
Comment (bimbolabuari) [inline README.md]: [ Optional] Why don't we add your social links, this helps recruiters to get in touch after checking out your project 😉

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hello Team, 

You have done a good job so far 🚀  

## Highlights 🎉 

- Installed React-Redux, Redux Logger, and React Router.
- Created routes and view components
-  Used `<NavLink />`  for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- Created directories for all Redux state slice files

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: README.md (hunk 2 above)
Comment (chuanqizhu0820) [reply README.md]: Thanks for the suggestion. Social links have been added.

File: src/components/Profile.js (hunk 1 above)
Comment (chuanqizhu0820) [reply src/components/Profile.js]: Thanks for the suggestion. All of them have been removed.

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi TEAM 👋🏽 , 

Good job implementing all the project requirements and the changes requested by the previous reviewer.
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
- [ ] Consider closing all the opened issues as by the completion of this project, you have resolved all the issues.

Cheers and Happy coding!👏👏👏

**Pull request merged**